### PR TITLE
Adding various improvements.

### PR DESCRIPTION
**Pull request by dreinsch**
- SC:
  - improved error handling
- Synth:
  - can now get synth control arguments
  - improved set syntax
- Buffer:
  - fixed some issues with multichannel buffers
  - added arguments to methods
- added SynthFamily
- fixed code style
- updated example notebooks

### CODE REVIEW

File: sc3nb/__init__.py [hunk 1]
```diff
@@ -8,4 +8,5 @@
 from .sc import startup, SC
 from .timed_queue import TimedQueue, TimedQueueSC
 from .buffer import Buffer
+from .synth import SynthDef, Synth, SynthFamily
```
Comment (thomas-hermann) [inline sc3nb/__init__.py:11]: We need to discuss SynthFamily, I think it should at least be renamed....

File: sc3nb/sc.py [hunk 2]
```diff
@@ -351,19 +354,24 @@ def exit(self):
...
         self.scp.kill()
+        self.scp.wait()
+        print(f"Done. ")
 
     def boot_with_blip(self):
         """Boots SuperCollider server with audio feedback
         """
 
-        print('Booting server...')
+        print('Booting scsynth...')
```
Comment (thomas-hermann) [inline sc3nb/sc.py:366]: actually, scsynth is the process, which features the server. As it is also said to boot the server and not booting scsynth in SC3, the wording is maybe misleading, I suggest 'Booting sc3 server...'

File: sc3nb/sc.py [hunk 3]
```diff
@@ -598,30 +618,36 @@ def __read_loop_unix(self, output, queue):
...
 
-    def SynthDef(self, **kwargs):
+    def SynthDef(self, *args, **kwargs):
         """
         Documentation see: :class:`SynthDef`
         """
-        return SynthDef(self, **kwargs)
+        return SynthDef(self, *args, **kwargs)
+
+    def SynthFamily(self, *args, **kwargs):
```
Comment (thomas-hermann) [inline sc3nb/sc.py:646]: We need to  discuss SynthFamily - I suggest to solve this differently for now

Comment (dreinsch) [reply sc3nb/sc.py]: I removed SynthFamily for now. If we feel the need to add something like this we can add it in another PR.

**Pull request merged**